### PR TITLE
GRD-72545-removing db instructions from plug-in readmes

### PR DESCRIPTION
**Pull request by AlexaNahum**
Signed-off-by: Alexa.Nahum <Alexa.Nahum@ibm.com>

### CODE REVIEW

File: docs/general topics/Configuring_data_source/aws_postgres.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+14. Select the type of Authentication for the database (choose from **Password Authentication**, **Password and IAM database authentication**, and **Password and Kerberos authentication**).
+15. Expand the Additional Configuration options:
+
+	a. Configure the **Database options**.
+
+	b. Select options for **Backup**.
+
+	c. If desired, enable encryption on the database instances.
+
+	d. In **Log exports**, select the Postgresql log type to publish to Amazon CloudWatch.
```
Comment (ofer-haim) [inline docs/general topics/Configuring_data_source/aws_postgres.md:26]: This is a part of the native-log instructions.. I think we should keep it in the main page too

File: docs/general topics/Configuring_data_source/azure_postresql.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+# Configuring the Azure PostgreSQL service
+
```
Comment (ofer-haim) [inline docs/general topics/Configuring_data_source/azure_postresql.md:2]: Logging..Important

File: docs/general topics/Configuring_data_source/gcp_bigquery.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+* In the Topic ID field, provide a unique topic name, for example, MyTopic.
+* Click ```Create Topic```.
+
+### Create a subscription in Pub/Sub
+* Display the menu for the topic created in the previous step and click ```New subscription```.
+* Type a name for the subscription, such as MySub.
+* Leave the delivery type as Pull.
+* Click ```Create```
+
+### Create a log sink in Pub/Sub
```
Comment (ofer-haim) [inline docs/general topics/Configuring_data_source/gcp_bigquery.md:15]: I think we need to keep this part in the plugin's readme

File: filter-plugin/logstash-filter-postgres-guardium/AwsPostgres_README.md [hunk 4]
```diff
@@ -1,62 +1,12 @@
...
-15. Expand the Additional Configuration options:
+## Enabling the PGAudit extension
 
-	a. Configure the database options.
-
-	b. Select options for Backup.
-
-	c. If desired, enable Encryption on the database instances.
-
-	d. In Log exports, select the Postgresql log type to publish to Amazon CloudWatch.
```
Comment (ofer-haim) [inline filter-plugin/logstash-filter-postgres-guardium/AwsPostgres_README.md:27]: We need this part in the plugin's readme

File: filter-plugin/logstash-filter-pubsub-bigquery-guardium/README.md [hunk 5]
```diff
@@ -43,65 +43,8 @@ roles/logging.privateLogViewer (Private Logs Viewer)
...
-* Sink name: Provide an identifier for the sink. Note that after you create the sink you cannot rename it. However, you can delete a sink and create a new one.
-* Sink description (optional): Describe the purpose or use case for the sink.
-* In the Sink destination panel, select the Pub/Sub topic as sink service and destination.
-* Choose logs to include in the sink in the Build inclusion filter panel.
-* You can filter the logs by log name, resource, and  severity.
-Multi-region
-* In cases of multiple regions, you need to do the same set of configurations per each region.
-Based on the region, different configuration files will be used for the input plug-in
-
-#### Set destination (TOPIC & SUBSCRIPTION) permissions
```
Comment (ofer-haim) [inline filter-plugin/logstash-filter-pubsub-bigquery-guardium/README.md:73]: We need to keep this part in the plugin's readme file

File: docs/general topics/Configuring_data_source/aws_postgres.md (hunk 1 above)
Comment (AlexaNahum) [reply docs/general topics/Configuring_data_source/aws_postgres.md:26]: only steps c and d? I'll try to add just those steps back in a way that makes sense.

File: docs/general topics/Configuring_data_source/azure_postresql.md (hunk 2 above)
Comment (AlexaNahum) [reply docs/general topics/Configuring_data_source/azure_postresql.md:2]: I put the entire thing back in. Let me know if I misunderstood.

File: filter-plugin/logstash-filter-postgres-guardium/AwsPostgres_README.md (hunk 4 above)
Comment (AlexaNahum) [reply filter-plugin/logstash-filter-postgres-guardium/AwsPostgres_README.md:27]: it looks like I corrected this in response to your first review comment. Correct me if I'm wrong.

File: filter-plugin/logstash-filter-pubsub-bigquery-guardium/README.md (hunk 5 above)
Comment (AlexaNahum) [reply filter-plugin/logstash-filter-pubsub-bigquery-guardium/README.md]: put back the "set destination (topic and subscription) permissions" section underneath the "create a log sink" section.

File: docs/general topics/Configuring_data_source/gcp_bigquery.md (hunk 3 above)
Comment (AlexaNahum) [reply docs/general topics/Configuring_data_source/gcp_bigquery.md:15]: @PratikshaSonawane can you please ensure that the flow makes sense in this page? https://github.com/AlexaNahum/universal-connectors/blob/main/filter-plugin/logstash-filter-pubsub-bigquery-guardium/README.md

File: filter-plugin/logstash-filter-pubsub-bigquery-guardium/README.md (hunk 5 above)
Comment (AlexaNahum) [reply filter-plugin/logstash-filter-pubsub-bigquery-guardium/README.md]: @PratikshaSonawane same as request above

File: docs/Guardium Data Protection/troubleshooting_tool.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
+# Troubleshooting tool
+
+The troubleshooting tool helps you identify and resolve issues in UC connections. The troubleshooting tool scans your Logstash log files and searches for known errors. Once it finds errors, it notifies you along with a description of the problem. . Additionally, the troubleshooting tool checks the status of Sniffer and Squid services and informs you if either of them is inactive.
+
+## How it works
+
+To avoid  old or irrelevent messages, the tool scans for errors in the Logstash logs file starting from the last time you ran troubleshooting.For example, if you last ran troubleshooting 3 hours ago, the troubleshooting tool will now only scan for errors from the past 3 hours. If this is your first time running the troubleshooting tool,  it will scan from the beginning of the Logstash logs file. 
+
+## Supported errors
```
Comment (ofer-haim) [inline docs/Guardium Data Protection/troubleshooting_tool.md:9]: I'm not sure this is something we want to specify..This is an internal list which we already updated (and will continue to do so)

Comment (AlexaNahum) [reply docs/Guardium Data Protection/troubleshooting_tool.md:5]: removed

File: docs/general topics/Configuring_data_source/gcp_bigquery.md (hunk 3 above)
Comment (AlexaNahum) [reply docs/general topics/Configuring_data_source/gcp_bigquery.md:15]: Pratiksha confirmed via Slack that the changes are ok

File: filter-plugin/logstash-filter-pubsub-bigquery-guardium/README.md (hunk 5 above)
Comment (AlexaNahum) [reply filter-plugin/logstash-filter-pubsub-bigquery-guardium/README.md]: Pratiksha confirmed via Slack that the changes are ok

**Pull request merged**